src/property/FaceliftProperty.h

```diff
@@ -433,16 +433,21 @@ class ModelProperty : public Model<ElementType>, public PropertyBase
 
     ModelProperty()
     {
+        QObject::connect(this, &facelift::ModelBase::beginInsertElements, this, [this] (int i1, int i2) {
```

Comment:
calling the paremeters "first" and "last" would be more consistent (like in Qt) and easier to comprehend.